students/piotrowski_stanislaw/lesson_03_functions/negative_power.py

```diff
@@ -0,0 +1,32 @@
+# https://snakify.org/lessons/functions/problems/negative_power/
+# piotrsta
+
+
+def power(a, n):
+    result = float(a) ** int(n)
+    return result
+
+
+def is_int(value):
+    try:
+        int(value)
+        return True
+    except ValueError:
+        return False
+
+
+def is_float(value):
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+a = input('Enter float value: ')
+n = input('Enter integer value: ')
+
+if is_float(a) and is_int(n):
```

Comment:
Wydaje mi się, że to moze byc troche przekombinowane rozwiązanie sprawdzania poprawności.
Wystarczyło by takie podejście jak użyłeś w `length_of_segment.py`
Zwłaszcza, że w wewnątrz funkcji `power` wykonujesz konwersje `int` i `float` a nie dodałes obsługi wyjątku w tym miejscu.
Jak możesz to popraw, we wszystkich zadaniach.